tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -166,11 +183,13 @@ public static void ClearProfilerEnvironmentVariables()
                 // Datadog
                 "DD_PROFILER_PROCESSES",
                 "DD_DOTNET_TRACER_HOME",
+                "DD_INTEGRATIONS",
```

Comment:
Did you intend to add DD_INTEGRATIONS and DD_TRACE_CALLTARGET_ENABLED back in? I thought we don't need them anymore

Reply:
Yes, call-site instrumentation was removed completely for v2.0, and call-target stopped using `integrations.json` a few releases ago.

FWIW, we had `DD_INTEGRATIONS` on this list until #2073 which cleaned up some leftover references. Probably not merged into this branch yet?

Reply:
Yeah, looks like a merging error

Reply:
Definite merging issue, will fix.